katello-agent/katello-agent.spec

```diff
@@ -88,13 +88,21 @@ cp extra/katello-agent-send.cron %{buildroot}%{_sysconfdir}/cron.d/%{name}
 rm -rf %{buildroot}
 
 %post
-chkconfig goferd on
 %if 0%{?fedora} > 18 || 0%{?rhel} > 6
-  /bin/systemctl start goferd > /dev/null 2>&1 || :
+  chkconfig goferd on
```

Comment:
Nitpick, If we're moving this into the systemd distro block, we should move to using `systemctl enable goferd`